tests/test_protocols/test_abci/helper.py

```diff
@@ -51,40 +49,15 @@
     Request,
     Response,
 )
-from packages.valory.connections.abci.tendermint.crypto import (  # type: ignore
-    keys_pb2,
-    proof_pb2,
-)
-from packages.valory.connections.abci.tendermint.types import (  # type: ignore
-    params_pb2,
-    types_pb2,
-)
 from packages.valory.protocols import abci as valory_abci_protocol
 from packages.valory.protocols.abci import AbciMessage
 from packages.valory.skills.abstract_round_abci.dialogues import AbciDialogues
 
 
 Node = Dict[str, Any]
-MODULE_PATH = Path(*tendermint.__package__.split(".")).absolute()
-python_files = tuple(MODULE_PATH.rglob("*/*.py"))
-keys = ["_".join(f.parts[-2:])[:-3] for f in python_files]
-
-
-def _exec_module_import(path: str) -> ModuleType:
-    """Execute the import of a module"""
-    spec = importlib.util.spec_from_file_location(path, path)
-    module = importlib.util.module_from_spec(spec)  # type: ignore
-    spec.loader.exec_module(module)  # type: ignore
-    return module
-
 
-def get_aea_type(data_type: str) -> str:
-    """Translate type to AEA-native ABCI format"""
-    if data_type in type_to_python:
-        if type_to_python[data_type] is None:
-            raise NotImplementedError(f"type_to_python: {data_type}")
-        return f"pt:{type_to_python[data_type].__name__}"
-    return f"ct:{data_type}"
+camel_to_snake = _camel_case_to_snake_case
+snake_to_camel = _to_camel_case
```

Comment:
Minor, but why not use `as` when importing?

Reply:
Could have indeed!